crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -316,59 +323,17 @@ mod tests {
         });
     }
 
-    /// There is a test that shows that if the receipt is set to complete
-    /// for a DHTOp we don't re-publish it
-    #[test_case(1, 1)]
-    #[test_case(1, 10)]
-    #[test_case(1, 100)]
-    #[test_case(10, 1)]
-    #[test_case(10, 10)]
-    #[test_case(10, 100)]
-    #[test_case(100, 1)]
-    #[test_case(100, 10)]
-    #[test_case(100, 100)]
-    fn test_no_republish(num_agents: u32, num_hash: u32) {
```

Comment:
Replaced by `retry_publish_until_receipts_received` which does not skip the step of publishing it once